File: Stage5/Java/Problem2908.java
package Stage5.Java;

import java.util.Scanner;

public class Problem2908 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String A = sc.next();
        String B = sc.next();

        int revA = Integer.parseInt(new StringBuilder(A).reverse().toString());
        int revB = Integer.parseInt(new StringBuilder(B).reverse().toString());

        System.out.println(Math.max(revA, revB));
        sc.close();
    }
}

File: Stage5/C/Problem2908.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>

void reverse(char* str) {
    int len = strlen(str);
    for (int i = 0; i < len / 2; i++) {
        char temp = str[i];
        str[i] = str[len - 1 - i];
        str[len - 1 - i] = temp;
    }
}

int main() {
    char A[4], B[4];
    scanf("%s %s", A, B);

    reverse(A);
    reverse(B);
    
    int revA = atoi(A);
    int revB = atoi(B);

    printf("%d\n", (revA > revB) ? revA : revB);
    return 0;
}